service/history/timerQueueProcessor.go

```diff
@@ -209,6 +212,15 @@ func (t *timerQueueProcessorImpl) NotifyNewTimer(timerTasks []persistence.Task)
 			t.minPendingTimer = ts
 			updatedMinTimer = true
 		}
+
+		switch task.GetType() {
+		case persistence.TaskTypeDecisionTimeout:
+			t.metricsClient.IncCounter(metrics.TimerTaskDecisionTimeoutScope, metrics.NewCounter)
```

Comment:
Ok with following this model, can you rename "new" to "new-timer" in that case ? Because with "new" I can't say whether its a new "timeout" or a new "timer" getting created for this timeout